enterprise/frontend/src/metabase-enterprise/clean_up/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { ModalRoute } from "metabase/hoc/ModalRoute";
+import { PLUGIN_COLLECTIONS } from "metabase/plugins";
+import { hasPremiumFeature } from "metabase-enterprise/settings";
+
+import { CleanupCollectionModal } from "./CleanupCollectionModal";
+
+// TODO: before merging, remove second conditional
```

Comment:
Commenting here for visibility - I don't know if you have a local git hook that makes it hard to commit TODOs

Reply:
this is going into john's branch. he hasn't added the feature yet, so he'll remove this once it's added.